digiwf-gateway/src/main/resources/application-local.yml

```diff
@@ -18,25 +12,29 @@ spring:
             allowedOrigins:
               - "http://localhost:8081"
               - "http://127.0.0.1:8081"
+              - "http://172.17.0.1"
+              - "http://host.docker.internal"
             allowedMethods: "*"
             allowedHeaders: "*"
             allowCredentials: true
             maxAge: 3600
       routes:
 
         - id: sso
-          uri: https://ssodev.muenchen.de/
+          uri: ${SSO_BASE_URL}
           predicates:
             - Path=/api/sso/userinfo
           filters:
-            - RewritePath=/api/sso/userinfo, /auth/realms/${spring.realm}/protocol/openid-connect/userinfo
+            - RewritePath=/api/sso/userinfo, /realms/${SSO_REALM}/protocol/openid-connect/userinfo
+
         - id: backend
           uri: http://localhost:39146/
           predicates:
             - Path=/api/digitalwf-backend-service/rest/**
           filters:
             - RewritePath=/api/digitalwf-backend-service/(?<urlsegments>.*), /$\{urlsegments}
             - RemoveResponseHeader=WWW-Authenticate
+
         - id: frontend
           uri: http://localhost:8081/
           predicates:
```

Comment:
If we use for the frontend and backend uri env variables we could remove the local and local-docker profile